code/game/machinery/bots/ed209bot.dm

```diff
@@ -754,14 +754,14 @@ Auto Patrol: []"},
 		if(perp.dna && perp.dna.mutantrace && perp.dna.mutantrace != "none")
 			threatcount += 2
 
-		if(!perp.wear_id)
+		if(!(perp.wear_id || perp.find_held_item_by_type(/obj/item/weapon/card/id))) //check for ID in either the ID slot or hands
 			if(idcheck)
 				threatcount += 4
 			else
 				threatcount += 2
 
 		//Agent cards lower threatlevel.
-		if(perp.wear_id && istype(perp.wear_id.GetID(), /obj/item/weapon/card/id/syndicate))
+		if(perp.wear_id && istype(perp.wear_id.GetID(), /obj/item/weapon/card/id/syndicate) || perp.find_held_item_by_type(/obj/item/weapon/card/id/syndicate) )
```

Comment:
For some reason it's not working when the ID is in the inactive hand now.
It works while the ID is in the pocket even, could the hand refactor have messed with the way that works?